content/docs/guide-development.md

```diff
@@ -117,7 +117,51 @@ https://renkulab.io/
 
 TODO: explain how to run server
 
+To support the development of workflows in Renku, a set of dedicated funcitonailities, provided as Renku plugins, are made available. In particular those aim to:
 
+* Offer an interactive (and non) visualization of the project Knowledge Graph (`renku-graph-vis` plugin)
```

Comment:
```suggestion
* Offer an interactive visualization of the project Knowledge Graph (`renku-graph-vis` plugin)
```

Reply:
the "and non" comes from the `display` functionality that outputs the png image

Reply:
true that